website/pages/docs/rules.zh-CN.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: 'Rules Of Blocks'
+description: 'How to use blocks in your projects the right way'
+---
+
+import { Callout } from 'nextra-theme-docs';
+import { AutomaticModeWarning } from '../../components/automatic-mode-warning';
+
+# 区块规则
+
+{/* <AutomaticModeWarning /> */}
+
+您来到这里可能是因为控制台中收到警告消息。
+
+<Callout type="warning">
+  <code><strong>\[Million.js\]</strong> 你做错事了！</code>
+
+  ```jsx showLineNumbers filename="App.jsx" {2}
+  <div>
+    <YourBlock>Uh oh...</YourBlock>
+              ^
+  </div>
+  ```
+</Callout>
+
+您可能会看到它的三个常见原因：
+
+1. 你可能 **违反了区块规则。**
+2.您可能 **忘记使用编译器。**
+3. 您已输入 **不受支持的行为。**
+
+## 打破区块规则
+
+您可能听说过 ["渐进增强,"](https://developer.mozilla.org/en-US/docs/Glossary/Progressive_Enhancement) 这个想法是工具根据支持的内容逐步使用功能。 类似地, Million.js 具有“渐进降级”的概念，即如果您使用不支持的功能，它将优雅地降级为默认的 React 渲染。
+
+本节重点介绍使用块时可能遇到的一些警告。请注意，这不是详尽的列表。
+
+<Callout type="info">
+  本节介绍使用块的惯用模式。它不是错误列表，如果您不遵循这些模式，您的应用程序仍然可以运行。
+</Callout>
+
+### 声明块
```

Comment:
```suggestion
### 声明 block
```
My suggestion is to use English to convey the technical emphasis you wish to describe, but in the singular form, as there is no plural concept in Chinese.